curriculum/challenges/english/25-front-end-development/workshop-loan-qualification-checker/66c8ba975ee7230e29f6c4aa.md

```diff
@@ -0,0 +1,33 @@
+---
+id: 66c8ba975ee7230e29f6c4aa
+title: Step 3
+challengeType: 1
+dashedName: step-3
+---
+
+# --description--
+
+Use `const` to create a `minIcomeForCar` and `minCreditScoreForCar` variables. Set them to `30000` and `650` respectively.
```

Comment:
same here.
they can use whatever (`var`, `let`, or `const`) to create the variables.